Makefile

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Do we run `make test` from the root directory or from the directory where the rego files are?